builder.sh

```diff
@@ -312,6 +323,9 @@ function run_build() {
             done
         done
     fi
+
+    # Singing image
+    codenotary_sign "${repository}/${image}:${version}"
```

Comment:
We should probably limit this to when `"$DOCKER_PUSH" == "true"` there is no need to sign if we are not pushing the image.

Reply:
that is

Reply:
![image](https://user-images.githubusercontent.com/15338540/112015671-5113b500-8b2c-11eb-8dc9-55a5990c8b05.png)

Reply:
oh.. my bad :D